sdk/identity/identity/src/msal/utils.ts

```diff
@@ -85,12 +85,14 @@ export function getKnownAuthorities(tenantId: string, authorityHost: string): st
  */
 export const defaultLoggerCallback: (
   logger: CredentialLogger,
+  allowPiiLogging?: boolean,
   platform?: "Node" | "Browser"
```

Comment:
Having two optional parameters is something we try to avoid. If there are more than 1 optional parameter, we prefer to use the options bag to avoid the case of passing undefined to the first one when we don;t want to set anything there, but do want to pass something to the second parameter.

In this case, do we even need the `platform` parameter if we can determine that using the `isNode` property?

Reply:
ahh correct! I like how you think :) I’ll clean this up!